### DISCUSSION TITLE
Block specific MAC addresses from joining the network

### DISCUSSION THREAD

**Discussion by riomccloud** [Mygod/VPNHotspot#416] [Ideas]
The current connection blocking feature is fine, but from what I've noticed the device keeps taking up a slot on the network. It would be interesting to have some way to prevent a specific device from connecting to the network.

I know that passwords exist, but in my case, some devices that I don't know end up entering the network thanks to some phones that allow password sharing (either by QR code or by viewing the key).

On second thought, a whitelist of MAC addresses would be a good solution too.

**Comment by Mygod** [comment]
This is supported on Android 11+ for system hotspot. You can configure it from the app by turning on "Control which client can use hotspot".

P.S. Also you should just change your password.
